src/wwwroot/services/auth.js

```diff
@@ -197,5 +198,25 @@
         }
       });
     }
+    function changePassword(old_pass, newPass, lang) {
+      return $http({
+        actionDescription: 'action_updating_password',
+        method: 'PUT',
+        url: RELAY_CONFIG.baseUrl + '/user/password/change?lang=' + lang,
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        data: {
+          "username": getUserName(),
+          "old_password": old_pass,
+          "password": newPass
+        }
+      })
+      .then(function (response) {
+        return response;
+      })
+      .catch(function (reason) {
```

Comment:
If the promise fails we're not doing anything. Is that OK? If so, why adding the catch function?